pydoctor/epydoc/markup/epytext.py

```diff
@@ -341,12 +344,12 @@ def parse(text: str, errors: Optional[List[ParseError]] = None) -> Optional[Elem
                         "epytext string.")
                 errors.append(StructuringError(estr, token.startline))
 
-    # If there was an error, then signal it!
-    if any(e.is_fatal() for e in errors):
-        if raise_on_error:
-            raise errors[0]
-        else:
-            return None
+    # If there was a fatal error, then raise exception!
+    # The exception was not raised by default leading into displaying literally nothing
+    # when an invalid epytext docstring was used. Now it will fall back to plaintext thanks to this exception.
+    for e in errors:
+        if e.is_fatal():
+            raise e
 
```

Comment:
This change is concurrent to the changes proposed in https://github.com/twisted/pydoctor/pull/407

Reply:
I should probably revert this changes and propose another PR for that or add a commit to #407. 
Please advise.

Reply:
Honestly, I hope we can merge this Pr without breaking it in several smaller pieces. 
I'm OK to remove some of the changes like this one, but removing all the cleanups regarding the imports and put them in a separate PR would be extra work for nothin gin my opinion. 

The diff is under 1000 lines, that is what the Twisted changes process advices. please don't ask me to change everything in this PR.

Reply:
I've reverted these changes.

Reply:
I'm not going to block this PR because it's changing too many things, but in general I'd be quicker and more motivated to review PRs that are smaller. So next time you notice that you'll need to do some cleanup before adding a feature, please create aim for multiple smaller PRs instead of one big one.

Think of Git history as a series of patches rather than a chronology of development. You can use `git rebase -i` to re-order commits, putting all the cleanups at the start of the chain. That way, it's easy to split them off into a separate branch later if necessary. If you do this while developing, you may have to occasionally do some conflict resolution, but it shouldn't be nearly as much work as doing it after all the code has been written.

Reply:
> So next time you notice that you'll need to do some cleanup before adding a feature, please aim for multiple smaller PRs instead of one big one.

Yes I Will try to do that. 

> You can use `git rebase -i` to re-order commits, putting all the cleanups at the start of the chain.

I'll try to get my head around git rebasing, it's still something that I'm not quite familiar with yet.